.github/workflows/publish-cdn.yml

```diff
@@ -0,0 +1,48 @@
+name: Publish CDN artifacts (versioned) to S3
+run-name: Publish CDN artifacts - versioned by @${{ github.actor }}
+
+on:
+  workflow_dispatch:
```

Comment:
wondering if this should be auto-deployed when `main` is updated, or keep it on dispatch. Thoughts @tay1orjones ?